tests/beamlines/unit_tests/test_beamline_utils.py

```diff
@@ -13,6 +13,14 @@
 from dodal.devices.zebra import Zebra
 from dodal.utils import make_all_devices
 
+from ...conftest import mock_beamline_module_filepaths
+
+
+@pytest.fixture
+def reset_i03():
+    beamline_utils.clear_devices()
+    mock_beamline_module_filepaths("i03", i03)
```

Comment:
I feel like using i03 as a default, especially when it overrides the check that BEAMLINE is set so it's possible and safe to connect to real hardware is going to cause trouble eventually.

Instead making clear_beamline_module_filepaths(), or set it to something completely impossible to collide with real beamline? And then the fixture can be "reset"

Reply:
This doesn't override any check on `BEAMLINE` as far as I can tell?

Reply:
Nevermind, I thought it was setting beamline_module.BL.